carbonmark-api/src/routes/projects/get.ts

```diff
@@ -136,12 +138,19 @@ const handler = (fastify: FastifyInstance) =>
       return composeOffsetProject(cmsProject, offset, lowestPrice);
     });
 
-    const filteredProjects = concat(projects, offsetProjects).filter(
-      ({ price }) => notNil(price) && parseFloat(price) > 0
-    );
+    const validProject = ({ price }: GetProjectResponse) =>
+      notNil(price) && parseFloat(price) > 0;
+
+    // Remove invalid projects and duplicates selecting the project with the lowest price
+    const filteredUniqueProjects = pipe(
+      concat,
+      filter(validProject),
+      sortBy("price"),
+      uniqBy(buildProjectKey)
+    )(projects, offsetProjects);
```

Comment:
should we just re-write our API in Elixir? xD
I'm kindof down...

Reply:
Ooh now we're talking